packages/stencil-component/src/components/read-along-component/read-along.tsx

```diff
@@ -1124,58 +1406,63 @@ export class ReadAlongComponent {
     return (
       <div id='read-along-container' class='read-along-container'>
         <h1 class="slot__header">
-          <slot name="read-along-header" />
```

Comment:
more mostly whitespace changes ... I changed the formatting too in dev.readalong-component, so this may cause some large problems